Mage/Task/BuiltIn/Deployment/Strategy/TarGzTask.php

```diff
@@ -132,7 +156,10 @@ public function run()
     protected function excludesListFile($excludesFile)
     {
         $excludesListFileRsync = '';
-        if (!empty($excludesFile) && file_exists($excludesFile) && is_file($excludesFile) && is_readable($excludesFile)) {
+        if (!empty($excludesFile)
+            && file_exists($excludesFile)
+            && is_file($excludesFile)
+            && is_readable($excludesFile)) {
```

Comment:
well since `is_readable` covers `file_exists` and `is_file`, I'd throw them out and leave only `is_readable` here because it also checks whether the file can be opened with read permissions.